src/json.hpp

```diff
@@ -144,6 +337,41 @@ struct DecimalSeparator : std::numpunct<char>
 
 }
 
+// taken from ranges-v3
+// TODO add doc
+template <typename T>
+struct _static_const
```

Comment:
I'm still confused by the "external identifier" thing, but it would surprise me if identifiers beginning with an underscore (except the global namespace) were to be reserved.

I would be an outlaw with all my prefixed private variables...